app/tests/unit/components/Form/ProjectForm.test.tsx

```diff
@@ -87,6 +87,7 @@ describe("The Project Form", () => {
       description: "d",
       operatorId: 1,
       fundingStreamId: 1,
+      fundingStreamRfpId: 1,
```

Comment:
I don't think that you need `fundingStreamId` in any of these `formData` anymore, only `fundingStreamRfpId`